templates/AureliaSpa/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,9 @@
+@{
+    ViewData["Title"] = "Home Page";
+}
+ 
+@section scripts {
```

Comment:
Any reason why the client should be doing 3 separate HTTP requests for these JS files? Couldn't the `bootstrap` and `modules` bundles be a single bundle? (Ideally, all the vendor dependencies would be in a single file)